docs/Installing.asciidoc

```diff
@@ -374,8 +374,10 @@ For a local instance setup you can simply execute the script:
 /usr/share/openqa/script/configure-web-proxy
 ----
 
-This will automatically setup a local Apache http proxy. Read on for more
-detailed setup instructions with all the details.
+This will automatically setup a local Apache http proxy. The script
+also supports NGINX and a custom port to listen on. Try `--help` to
```

Comment:
```suggestion
also supports nginx and a custom port to listen on. Try `--help` to
```

as written on https://nginx.org/en/

Reply:
We use NGINX in our docs, which is the other option